app/src/main/java/org/openobservatory/netprobe/utils/Alert.java

```diff
@@ -52,4 +52,57 @@ public void onClick(DialogInterface dialog, int id) {
 
                 }).show();
     }
+
+    public static String formatString(String text){
```

Comment:
We don't need it anymore, that code is going to be removed with the new ooni-wui